storage_manager_new.py

```diff
@@ -43,14 +43,12 @@ def show_all_items(self):
             print('---\nList of items is empty\n---')
         print('\n')
 
-    def show_items_from_category(self):
-        category = input(
-            'Print items from category:\n').lower()
-        print('')
+    def find_items_from_category(self, category):
+        items_from_specified_category = []
         for item in self.items:
-            if category == item.category.lower():
-                print(f'({item.category})  {item.type} {item.model}')
-        print('\n')
+            if category.lower().strip() == item.category.lower():
+                items_from_specified_category.append(item)
+        return items_from_specified_category
```

Comment:
Not saying it is necessary here but you could do all of it in single line `return [item for item in self.items if category.lower().strip() == item.category.lower()]` :)